ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-telemetry-postcard-policy.j2

```diff
@@ -0,0 +1,69 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor telemetry postcard policy #}
+{% if monitor_telemetry_postcard_policy is arista.avd.defined %}
+!
+monitor telemetry postcard policy
+{%     if monitor_telemetry_postcard_policy.disabled is arista.avd.defined(false) %}
+   no disabled
+{%     elif monitor_telemetry_postcard_policy.disabled is arista.avd.defined(true) %}
+   disabled
+{%     endif %}
+{%     if monitor_telemetry_postcard_policy.ingress.sample.rate is arista.avd.defined %}
+   ingress sample rate {{ monitor_telemetry_postcard_policy.ingress.sample.rate }}
+{%     elif monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.value is arista.avd.defined and monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.mask is arista.avd.defined %}
+   ingress sample tcp-udp-checksum value {{ monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.value }} mask {{ monitor_telemetry_postcard_policy.ingress.sample.tcp_udp_checksum.mask }}
+{%     endif %}
+{%     if monitor_telemetry_postcard_policy.marker_vxlan.enabled is arista.avd.defined(true) %}
+   marker vxlan
+{%     elif monitor_telemetry_postcard_policy.marker_vxlan.header_word_zero_bit is arista.avd.defined %}
+   marker vxlan header word 0 bit {{ monitor_telemetry_postcard_policy.marker_vxlan.header_word_zero_bit }}
+{%     endif %}
```

Comment:
I think we want the enabled to be needed and then optionally adding the header word. That is more in line with other data models.

```suggestion
{%     if monitor_telemetry_postcard_policy.marker_vxlan.enabled is arista.avd.defined(true) %}
{%         set marker_cli = "marker vxlan" %}
{%         if monitor_telemetry_postcard_policy.marker_vxlan.header_word_zero_bit is arista.avd.defined %}
{%             set marker_cli = marker_cli ~ " header word 0 bit " ~ monitor_telemetry_postcard_policy.marker_vxlan.header_word_zero_bit %}
{%         endif %}
{{ marker_cli }}
{%     endif %}
```

Reply:
Fixed.